plugins/metadata/plugin.go

```diff
@@ -132,21 +151,18 @@ func init() {
 					}
 
 					log.G(ic.Context).WithField("policy", cfg.ContentSharingPolicy).Info("metadata content store policy set")
+
+					if cfg.NoSync {
+						options.NoSync = true
+						options.NoGrowSync = true
+					}
 				}
 			}
+			log.G(ic.Context).WithField("plugin", "bolt").Infof("bolt config: %+v", options)
```

Comment:
Maybe just log a warning in the new sync case

Reply:
Updated